Lib/test/test_capi/test_unicode.py

```diff
@@ -1297,6 +1297,107 @@ def test_comparewithasciistring(self):
         # CRASHES comparewithasciistring([], b'abc')
         # CRASHES comparewithasciistring(NULL, b'abc')
 
+    @support.cpython_only
+    @unittest.skipIf(_testcapi is None, 'need _testcapi module')
+    def test_equaltoutf8(self):
+        """Test PyUnicode_EqualToUTF8()"""
```

Comment:
```suggestion
```

Reply:
@erlend-aasland: why do you want to remove these docs? they are displayed when tests are run in verbose mode.

Reply:
yes, they clutter the verbose mode run, and they do not add any information; it is just a repetition of the test method name, just with a slightly different formatting. The test method name is already shown when run in verbose mode; I do not need to read it twice.

Reply:
If the test method name is not understandable, the solution is to choose a better test method name. In this case `test_pyunicode_equaltoutf8` or `test_PyUnicode_EqualToUTF8` are better solutions to the problem.